validations/validate-login.js

```diff
@@ -0,0 +1,18 @@
+import Joi from '@hapi/joi';
+
+const validateLogin = (user) => {
+  const schema = Joi.object().keys({
```

Comment:
nice. now you should use this in a middleware

Reply:
done sir